diff --git a/dist/chunk-snapshot-manager.70695b70.js b/dist/chunk-snapshot-manager.70695b70.js
index v0.26.3..v0.26.3 100644
--- a/dist/chunk-snapshot-manager.70695b70.js
+++ b/dist/chunk-snapshot-manager.70695b70.js
@@ -8044,7 +8044,7 @@ class DefaultReporter extends BaseReporter {
     super.onWatcherStart();
   }
   onCollected() {
-    if (this.isTTY) {
+    if (this.isTTY || this.mode === "typecheck") {
       this.rendererOptions.logger = this.ctx.logger;
       this.rendererOptions.showHeap = this.ctx.config.logHeapUsage;
       this.rendererOptions.mode = this.mode;
