diff --git a/dist/client/theme-default/components/VPAlgoliaSearchBox.vue b/dist/client/theme-default/components/VPAlgoliaSearchBox.vue
index v1.0.0-alpha.26..v1.0.0-alpha.26 100644
--- a/dist/client/theme-default/components/VPAlgoliaSearchBox.vue
+++ b/dist/client/theme-default/components/VPAlgoliaSearchBox.vue
@@ -70,7 +70,7 @@
         ref: undefined,
         constructor: undefined,
         key: undefined,
-        props: { href: hit.url, children }
+        props: { children, href: hit.url, rel: 'noopener', target: '_blank' }
       }
     }
   })
diff --git a/dist/client/theme-default/components/VPDocFooter.vue b/dist/client/theme-default/components/VPDocFooter.vue
index v1.0.0-alpha.26..v1.0.0-alpha.26 100644
--- a/dist/client/theme-default/components/VPDocFooter.vue
+++ b/dist/client/theme-default/components/VPDocFooter.vue
@@ -43,13 +43,13 @@
       <div class="pager">
         <a v-if="control.prev" class="pager-link prev" :href="normalizeLink(control.prev.link)">
           <span class="desc">{{ theme.docFooter?.prev ?? 'Previous page' }}</span>
-          <span class="title">{{ control.prev.text }} </span>
+          <span class="title" v-html="control.prev.text"></span>
         </a>
       </div>
       <div class="pager" :class="{ 'has-prev': control.prev }">
         <a v-if="control.next" class="pager-link next" :href="normalizeLink(control.next.link)">
           <span class="desc">{{ theme.docFooter?.next ?? 'Next page' }}</span>
-          <span class="title">{{ control.next.text }}</span>
+          <span class="title" v-html="control.next.text"></span>
         </a>
       </div>
     </div>
